timedomain/sp_utils.py

```diff
@@ -137,22 +176,31 @@ def postDESIInstrument():
     def test():
 
         
-#         response = SkyPortal.api('DELETE', '{}/api/candidates/{}'.format(SkyPortal.url,"35191288252861933"))
+        response = SkyPortal.api('DELETE', '{}/api/candidates/{}'.format(SkyPortal.url,"35191288252861933"))
```

Comment:
In the end you are not really working with test(), so don't make change modifications to it.